Essentials/src/com/earth2me/essentials/commands/Commandmore.java

```diff
@@ -24,14 +24,34 @@ public void run(final Server server, final User user, final String commandLabel,
             throw new Exception(tl("fullStack"));
         }
         final String itemname = stack.getType().toString().toLowerCase(Locale.ENGLISH).replace("_", "");
-        if (ess.getSettings().permissionBasedItemSpawn() ? (!user.isAuthorized("essentials.itemspawn.item-all") && !user.isAuthorized("essentials.itemspawn.item-" + itemname) && !user.isAuthorized("essentials.itemspawn.item-" + stack.getTypeId())) : (!user.isAuthorized("essentials.itemspawn.exempt") && !user.canSpawnItem(stack.getTypeId()))) {
+        if (!user.canSpawnItem(stack.getType())) {
             throw new Exception(tl("cantSpawnItem", itemname));
         }
-        if (user.isAuthorized("essentials.oversizedstacks")) {
-            stack.setAmount(ess.getSettings().getOversizedStackSize());
+        if (args.length > 1) {
+            int newAmount = stack.getAmount();
+
+            try {
+                newAmount += Integer.parseInt(ChatColor.stripColor(args[1]));
```

Comment:
Why would we need to strip color from this argument?

Reply:
I don't know exactly how `parseInt` functions but I assumed it would fail if it received something with a formatting code. We strip the colour from the argument so that it will work even if a formatting code is present.

Reply:
Why would a client be sending us an argument for a number with color? Its a good thing if it fails - what the client has sent us is not a number - it is a formatting code plus a number.

Reply:
You're right, I suppose it would be a better idea to reject all but raw numbers.